### DISCUSSION TITLE
Mac apps on cli ?

### DISCUSSION THREAD

**Discussion by jhay06** [ravynsoft/ravynos#385] [Q&A]
Hi , 
  just want to ask ravyn os can run  mac apps with cli ? example 

xcodebuild 
homebrew
mac ports 

etc.

Thanks

**Comment by ashifolfi** [comment]
Apps compiled for macos won't work right now. They have to be recompiled for ravynos. That being said we're also lacking a lot of functionality in our own frameworks so there's a very high chance that mac apps won't run period due to missing calls even when recompile attempts are made.
